### PR TITLE
README: Update for FCCT -> Butane rename

### PR DESCRIPTION
**Pull request by travier**
Also update the docs to match README content.

### CODE REVIEW

**Comment by travier** [comment]
I have not yet looked at the code if more changes are needed or beneficial.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # terraform-provider-ct [![Build Status](https://github.com/poseidon/terraform-provider-ct/workflows/test/badge.svg)](https://github.com/poseidon/terraform-provider-ct/actions?query=workflow%3Atest+branch%3Amaster)
 
-`terraform-provider-ct` allows Terraform to validate a [Container Linux Config](https://github.com/coreos/container-linux-config-transpiler/blob/master/doc/configuration.md) or [Fedora CoreOS Config](https://github.com/coreos/fcct/blob/master/docs/configuration-v1_1.md) and transpile it as [Ignition](https://github.com/coreos/ignition) for machine consumption.
+`terraform-provider-ct` allows Terraform to validate a [Container Linux Config](https://github.com/coreos/container-linux-config-transpiler/blob/master/doc/configuration.md) or a [Butane config](https://coreos.github.io/butane/specs/) and transpile it as an [Ignition config](https://github.com/coreos/ignition) for machine consumption.
```
Comment (bgilbert) [inline README.md:3]: Let's link to https://coreos.github.io/ignition/ while we're here.

File: README.md [hunk 2]
```diff
@@ -74,23 +74,23 @@ $ terraform init
 
 ## Versions
 
-Fedora CoreOS Config's contain a `version` that is associated with an Ignition format verison. For example, a FCC with `version: 1.0.0` would produce Ignition `3.0.0`, across future releases.
+Butane configs contain a `version` that is associated with an Ignition format verison. For example, a Butane config with `version: 1.0.0` would produce an Ignition config with version `3.0.0`, across future releases.
```
Comment (bgilbert) [inline README.md:77]: s/verison/version/

File: docs/index.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # config-transpiler Provider
 
-`terraform-provider-ct` allows Terraform to validate a [Container Linux Config](https://github.com/coreos/container-linux-config-transpiler/blob/master/doc/configuration.md) or [Fedora CoreOS Config](https://github.com/coreos/fcct/blob/master/docs/configuration-v1_2.md) and transpile it as [Ignition](https://github.com/coreos/ignition) for machine consumption.
+`terraform-provider-ct` allows Terraform to validate a [Container Linux Config](https://github.com/coreos/container-linux-config-transpiler/blob/master/doc/configuration.md) or a [Butane config](https://coreos.github.io/butane/specs/) and transpile it as an [Ignition config](https://github.com/coreos/ignition) for machine consumption.
```
Comment (bgilbert) [inline docs/index.md:3]: https://coreos.github.io/ignition/ here too.

**Comment by travier** [comment]
Thanks, updated. This should be ready as this is docs only. Will investigate the code later.

**Comment by dghubble** [comment]
Source code uses the new butane package naming

**Pull request merged**